src/shaping/shaping_linux.go

```diff
@@ -104,10 +107,42 @@ func (nl *netlinkShaper) DeleteGroup(id int64) error {
 }
 
 func (nl *netlinkShaper) Shape(id int64, shaping *atc_thrift.Shaping) error {
-	// Root qdisc: (already exists!)
-	//qdisc htb 1: root refcnt 2 r2q 10 default 0 direct_packets_stat 37016
+	wan, err := netlink.LinkByName(LAN_INT)
+	if err != nil {
+		return fmt.Errorf("Could not find lan(%s) interface: %v", LAN_INT, err)
+	}
+	lan, err := netlink.LinkByName(WAN_INT)
+	if err != nil {
+		return fmt.Errorf("Could not find wan(%s) interface: %v", WAN_INT, err)
+	}
```

Comment:
We could do this lookup statically in initialization, but the performance gain is minimal, and it means bouncing the interfaces requires a restart of atcd, which is one thing I don't like about the current python implementation.